yotiprofile.go

```diff
@@ -164,12 +124,65 @@ func (p Profile) DocumentImages() (*attribute.ImageSliceAttribute, error) {
 	return nil, nil
 }
 
+// ApplicationName is the name of the application
+func (p ApplicationProfile) ApplicationName() *attribute.StringAttribute {
+	return p.GetStringAttribute(attrConstApplicationName)
+}
+
+// ApplicationURL is the URL where the application is available at
+func (p ApplicationProfile) ApplicationURL() *attribute.StringAttribute {
+	return p.GetStringAttribute(attrConstApplicationURL)
+}
+
+// ApplicationReceiptBgColor is the background colour that will be displayed on
+// each receipt the user gets as a result of a sharing with the application.
+func (p ApplicationProfile) ApplicationReceiptBgColor() *attribute.StringAttribute {
+	return p.GetStringAttribute(attrConstApplicationReceiptBGColor)
+}
+
+// ApplicationLogo is the logo of the application that will be displayed to
+// those users that perform a sharing with it.
+func (p ApplicationProfile) ApplicationLogo() *attribute.ImageAttribute {
+	return p.GetImageAttribute(attrConstApplicationLogo)
+}
+
 // GetAttribute retrieve an attribute by name on the Yoti profile. Will return nil if attribute is not present.
-func (p Profile) GetAttribute(attributeName string) *attribute.GenericAttribute {
+func (p baseProfile) GetAttribute(attributeName string) *attribute.GenericAttribute {
 	for _, a := range p.attributeSlice {
 		if a.Name == attributeName {
 			return attribute.NewGeneric(a)
 		}
 	}
 	return nil
 }
+
+func (p baseProfile) GetStringAttribute(attributeName string) *attribute.StringAttribute {
```

Comment:
Should these methods be exported, or can they be private?

Reply:
I missed that this is to allow getting various attribute types by name that don't have a getter - will resolve